### PR TITLE
Removed css import in index.js

### PR DESCRIPTION
**Pull request by BossBele**
This is required for nextJS support:

```
./node_modules/react-gauge-chart/dist/GaugeChart/style.css
Global CSS cannot be imported from within node_modules.
Read more: https://err.sh/next.js/css-npm
Location: node_modules/react-gauge-chart/dist/GaugeChart/index.js
```
Perhaps this is not the best solution, but it is a temporary solution to this problem. Please find a way to fix this. The community at nextJS needs it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/Martin36/react-gauge-chart"
+    "url": "https://github.com/BossBele/react-gauge-chart"
```
Comment (Martin36) [inline package.json:21]: Leave this as it were

File: dist/GaugeChart/index.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ var _d = require("d3");
 
 var _propTypes = _interopRequireDefault(require("prop-types"));
 
-require("./style.css");
+// require("./style.css");
```
Comment (Martin36) [inline dist/GaugeChart/index.js:14]: Why did you remove this? Won't that affect the styling?

**Review by Martin36 — CHANGES_REQUESTED** [review]
See my comments

File: package.json (hunk 1 above)
Comment (BossBele) [reply package.json:21]: Okay

File: dist/GaugeChart/index.js (hunk 2 above)
Comment (BossBele) [reply dist/GaugeChart/index.js:14]: I removed it since nextjs restricts css imports in modules. As I tested it, it does not affect styling.
However, to solve this and ensure that the styles are not tempered with, include the css in the index.js file as CSS in JS. Allow me to update the pull request.

File: package.json (hunk 1 above)
Comment (BossBele) [reply package.json]: Did

File: dist/GaugeChart/index.js (hunk 2 above)
Comment (BossBele) [reply dist/GaugeChart/index.js]: Added style in src/lib/GaugeChart/index.js

**Comment by BossBele** [comment]
Edited src/lib/GaugeChart/index.js and removed src/lib/GaugeChart/index.css
Please Review
